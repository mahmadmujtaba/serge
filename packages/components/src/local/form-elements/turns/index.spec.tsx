/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import Turns from './index'

it('Turns renders correctly', () => {
  const tree = renderer
    .create(<Turns />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
