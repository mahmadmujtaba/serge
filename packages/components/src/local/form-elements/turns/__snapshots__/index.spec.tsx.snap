// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Turns renders correctly 1`] = `
<section
  className="input-container"
>
  <h2>
    For
  </h2>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
        value={5}
      />
    </div>
  </div>
   Turns
</section>
`;
